linkerd/app/inbound/src/lib.rs

```diff
@@ -243,7 +247,14 @@ impl Config {
                     .into_inner(),
             ))
             .push_map_target(endpoint::Logical::from)
-            .push(profiles::discover::layer(profiles_client))
+            // TODO: Profile resolution should be time-bounded so that slowness
+            // cannot cause inbound requests to fail.
```

Comment:
shall we merge #694 first and use that here, or should we do that as a follow-up? probably fine either way?

Reply:
i think it's fine either way. we'll need both in before the next release, but ordering doesn't really matter.